lesson10/main.js

```diff
@@ -50,9 +50,16 @@ window.onload = () => {
   };
 
   const callLists = async () => {
-    const listsContents = await fetchedData;
-    loaded();
-    createNewList(listsContents);
+    try {
+      const listsContents = await fetchedData;
+      createNewList(listsContents);
+    } catch (error) {
```

Comment:
IMO

I've been told by Mr.Morita the following

Do not execute "createNewList" in the try.

This is because you can’t judge
whether the error is from "fetchedData" or "createNewList".

Even if an error occurs from "createNewList", 
"データの取得ができませんでした。" will be displayed.

Reply:
@matsuwork
Thank you for your advice.I did't realized that but that's a good point!
I searched how to solve and I got a hint. Here’s what I got, I'm not sure it's correct though.

- Implemented return in the try block
- moved `createNewList` from try block to global and executed with then method

[d217c13 ](https://github.com/yuka830/js-lessons/pull/12/commits/d217c1379d661eeb4196f1977704d15c36bf8008)
[CodeSandbox](https://codesandbox.io/s/lesson10-gn6i7?file=/main.js)

Reply:
The idea is correct.
Async await is recommended,
so I think it would be better to use that.

Reply:
@matsuwork
Thank you for the reply. I rewrote `createNewList` part from then method to async/await.
[4664130](https://github.com/yuka830/js-lessons/pull/12/commits/46641304373e21b99f09adef43bad8a946620735)